openml/flows/flow.py

```diff
@@ -122,3 +407,68 @@ def _get_name(self):
         return self.name
 
 
+def create_flow_from_model(model, converter, description=None):
+    """Use a converter to create an OpenMLFlow from model.
+
+    Allows to configure how a model (for example a scikit-learn estimator) is
+    transformed into an OpenMLFlow.
+
+    Parameters
+    ----------
+    model : object
+        ML model. Must match the converter.
+    converter : object
+        Class that implements a method `flow = serialize_object(model)`.
```

Comment:
When reading this code it might seem to be too complicated and general for a first PR, but writing coding with this level of indirection actually helped me a lot coding this PR. Also, it more or less separates the tests of both. In the long run, this indirection allows developers of other toolboxes (Orange, Keras?) to easily plug in a converter (although this would fail when someone tries to run the flow, but we can solve that later).